build.sbt

```diff
@@ -0,0 +1,184 @@
+// Configurable settings
```

Comment:
No need of sbt files. Let's just remove the sbt stuff. Things should be built in Mill.